apps/web/components/MailingListSignup/MailingListSignup.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState, useEffect } from 'react'
+import { useFormik } from 'formik'
+import jsonp from 'jsonp'
+import { NewsletterSignup } from '@island.is/island-ui/core'
+import { isValidEmail } from '@island.is/web/utils/isValidEmail'
+import { GetNamespaceQuery } from '@island.is/web/graphql/schema'
+import { useNamespace } from '@island.is/web/hooks'
+
+type FormState = {
+  type: 'default' | 'error' | 'success'
+  message: string
+}
+
+interface FormProps {
+  email: string
+}
+
+interface MailingListSignupProps {
+  namespace: GetNamespaceQuery['getNamespace']
+  id: string
+  title: string
+  description: string
+  inputLabel: string
+  placeholder?: string
+  buttonText: string
+  mailingListUrl: string
+}
+
+export const MailingListSignup: React.FC<MailingListSignupProps> = ({
+  namespace,
+  id,
+  title,
+  description,
+  inputLabel,
+  placeholder,
+  buttonText,
+  mailingListUrl,
+}) => {
+  // Validate Mailing List Url contains email replacement token
+  // TODO: What is the proper way of logging such validation errors?
```

Comment:
@ingolfurs @Ingimarsson 
What is the proper way of logging validation errors, such as this one, where a settings string from Contentful is being validated? (Currently, I'm simply logging a warning to console.)